curriculum/challenges/english/21-a2-english-for-developers/learn-how-to-ask-and-share-about-educational-and-professional-background/657b138d33db9e50f116b5f0.md

```diff
@@ -0,0 +1,56 @@
+---
+id: 657b138d33db9e50f116b5f0
+title: Task 15
+challengeType: 19
+dashedName: task-15
+audioPath: curriculum/js-music-player/We-Are-Going-to-Make-it.mp3
+---
+
+<!--
+AUDIO REFERENCE:
+Anna: Great! Can you tell me about your qualifications? Were you educated in computer science?
+-->
+
+# --description--
+
+In English, when asking about past events or experiences, you often use the past simple interrogative. It begins with a past tense verb (like `Were`), followed by the subject (like `you`) and the rest of the sentence.
+
+For example: `Were you at the meeting yesterday?`
```

Comment:
```suggestion
Listen to the audio and answer the question.
```
This explanation was given in another challenge before. That's why I'm deleting it.